Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteScrollingCoordinatorProxyIOS.mm

```diff
@@ -369,15 +374,57 @@
     return activePoint;
 }
 
+void RemoteScrollingCoordinatorProxyIOS::displayDidRefresh(PlatformDisplayID displayID)
+{
+#if ENABLE(THREADED_ANIMATION_RESOLUTION)
+    updateAnimations();
+#endif
+}
+
+RemoteLayerTreeDrawingAreaProxyIOS& RemoteScrollingCoordinatorProxyIOS::drawingAreaIOS() const
+{
+    auto* drawingArea = dynamicDowncast<RemoteLayerTreeDrawingAreaProxy>(webPageProxy().drawingArea());
```

Comment:
Why not just add support for `dynamicDowncast<RemoteLayerTreeDrawingAreaProxyIOS>` ?

Reply:
Specialized traits for `DrawingArea` use the `DrawingAreaType` enum which does not contain a specific type for the Mac and iOS variants of `RemoteLayerTreeDrawingAreaProxy`. I'm not sure it's worth changing this system for what is only private use within `RemoteScrollingCoordinatorProxyIOS`.